src/dataDisplay/icon/iconConstants.js

```diff
@@ -1,6 +1,81 @@
 import PropTypes from 'prop-types';
+import Utils from '../../utils/utils';
 
-export const TITLE_PROP_TYPE = PropTypes.oneOfType([
+const titlePropType = PropTypes.oneOfType([
     PropTypes.string,
     PropTypes.bool,
 ]);
```

Comment:
**_Not a Blocker_**
Just wondering why separate this out versus define it inline with the others.

Reply:
I don't know why. I thought the same thing, but never came back to it.

Reply:
**_Not a Blocker_**
We can move it inline, or we can leave it as-is.  Your call.